packages/notification-services-controller/package.json

```diff
@@ -67,6 +67,16 @@
         "default": "./dist/NotificationServicesPushController/index.cjs"
       }
     },
+    "./push-services/web": {
```

Comment:
exposing new subpath exports to differ between web and mobile specific helpers.

Reply:
This can be done iteratively. We have done web, since there was existing code that we refactored and gutted from this controller.

We will do Mobile after PoC'ing this and seeing if there is value from it.

Reply:
Also to note that these subpaths are fully isolated and not used anywhere, so they should be tree-shakeable for bundle splitting.

However specific modules for web/mobile might run into lavamoat issues... or maybe not... we'll see.

Reply:
Hmm audit tools and lavamoat cannot distinguish this so it looks like it web is being used in mobile...
Meh we can refactor this to be only in extension in near future.